harmony_regridding_service/grid.py

```diff
@@ -251,16 +303,11 @@ def create_area_definition_for_projected_source_grid(
     filepath: str,
     dimension_pair: tuple[str, str],
     var_info: VarInfoFromNetCDF4,
-    override_crs: CRS | None = None,
```

Comment:
Is this removal just because that override isn't being sent along in either of the places this function is called? I'm not sure I am following from the old documentation string comment when/where they might have come from. (Maybe the configuration file?)

Reply:
You're in the wrong PR.  This is my PR to the actual work, and this override_crs just didn't work and is totally wrong.

Reply:
Okay. I'll continue looking through the longer one.